src/main/java/Duke.java

```diff
@@ -1,10 +1,38 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        System.out.println("Hello! I'm Duke");
+        System.out.println("What can I do for you?");
+        Task[] list = new Task[100];
```

Comment:
Should the size 100 be extracted out as a constant?

Reply:
Yes, I have done this but did not push yet. thanks for the feedback!